docs/css/style-v18.css

```diff
@@ -164,6 +164,20 @@ ul.sidenav code {
     box-shadow: none;
 }
 
+.switch label input[type=checkbox]:checked+.lever {
```

Comment:
The new style looks really nice, thanks.

But... shouldn't you create a `style-v19.css` file instead? :roll_eyes:

Reply:
You're right! It's now fixed.